### ISSUE TITLE
Homebrew Issue: Library not loaded openssl@1.1

### ISSUE THREAD

**Issue by jonas-tm** [vitobotta/hetzner-k3s#274]
Hi, first of all thanks for providing this awesome tool.

I recently reset my Mac (ARM, Ventura 13.5) and freshly installed hetzner-k3s via homebrew.
Unfortunately I facing the following issue when running it:

```bash
dyld[73707]: Library not loaded: /opt/homebrew/opt/openssl@1.1/lib/libssl.1.1.dylib
  Referenced from: <24E7E08F-3DC4-35D7-829E-D3DC1572649C> /opt/homebrew/Cellar/hetzner_k3s/1.1.3/bin/hetzner-k3s
  Reason: tried: '/opt/homebrew/opt/openssl@1.1/lib/libssl.1.1.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/openssl@1.1/lib/libssl.1.1.dylib' (no such file), '/opt/homebrew/opt/openssl@1.1/lib/libssl.1.1.dylib' (no such file), '/usr/local/lib/libssl.1.1.dylib' (no such file), '/usr/lib/libssl.1.1.dylib' (no such file, not in dyld cache)
zsh: abort      hetzner-k3s
```

Any idea what could be the issue?

**Comment by vitobotta** [comment]
Hi, looks like a problem with your homebrew setup. Can you try reinstalling openssl? Also try `brew doctor` as it may point out problems with the installation.

**Comment by jonas-tm** [comment]
I did both already. 
However seems like my homebrew installed `openssl@3.1.2` instead of the `1.1` linked in the error message.

**Comment by vitobotta** [comment]
Did you try installing 1.1?

**Comment by jonas-tm** [comment]
Yes this did the trick. However would expect that homebrew would install the correct version automatically when install hetzner-k3s. Is there maybe an adjustment to the hombrew formula needed?

**Comment by vitobotta** [comment]
I'll take a look at the formula but glad it's working now :)

**Comment by vitobotta** [comment]
Hi @jonas-tm I released 1.1.4 now and also updated the formula.
